src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PATH;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+import com.opencsv.CSVWriter;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+/**
+ * The type Export command.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Exports person data to a CSV file.\n"
+        + "Parameters: FILE_PATH\n"
+        + "[" + PREFIX_PATH + "NAME] "
+        + "Example: " + COMMAND_WORD + " data/persons.csv";
+
+    private final String filePath;
+
+    public ExportCommand(String filePath) {
+        this.filePath = filePath;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        List<Person> persons = model.getFilteredPersonList();
```

Comment:
This only gets the contacts are currently displayed on the list and export will only export those rather than the whole address book.